docker/Dockerfile.x86_64-unknown-linux-gnu.centos

```diff
@@ -26,5 +26,6 @@ COPY --from=0 /qemu /qemu
 
 COPY linux-runner base-runner.sh /
 
+ENV CROSS_TOOLCHAIN_PREFIX=
```

Comment:
ditto, do we need this?

Reply:
It ensures consistency in all cases, and makes sure that scripts with `set -u` won't fail. It might not be technically needed, but ensures we have the exact same behavior for all image types. The only somewhat difference is for `emscripten` images, but that's because we really can't have a viable prefix for them, so we just use `em`.